routes/api/user.js

```diff
@@ -32,6 +32,7 @@ router.route('/:id')
     })
     .patch(async (req, res) => {
         try {
+            console.log(req.body);
```

Comment:
you can remove this or add more information/context if you are still using it for debugging purposes/logging.  for example,
```suggestion
            console.log(`[DEBUG] PATCH /users/${id} :: ${req.body}`);
```
or something like that, that gives the reader of this log context about where it is coming from.  I guess if you have morgan configured the context could be gleamed from the surrounding middleware logs but still 😊